saleor/core/db_routers.py

```diff
@@ -0,0 +1,18 @@
+from django.conf import settings
+
+
+class PrimaryReplicaRouter:
+    def db_for_read(self, model, **hints):
+        """Read from replica."""
+        return settings.DATABASE_CONNECTION_REPLICA_NAME
```

Comment:
We can't implement this method. We don't want to read from a replica during mutations, especially not inside transactions when we do `SELECT ... FOR UPDATE`.

Reply:
Removed.